pagopa/pn-frontend
# Added width limit for file names in notification details documents with ellipsis.

## Short description
Added width limit for file names in notification details documents with ellipsis.

## List of changes proposed in this pull request
- File name in NotificationDetailsDocuments have a width limit and it will show ellipsis if name exceed the width

## How to test
Launch app PF in DEV, login with ada and search for iun AVRA-AEZY-HGUW-202303-J-1
Then look in attached documents box for a long file name. You should see ellipsis at the end of the truncated file name now.
